corehq/apps/api/resources/v0_5.py

```diff
@@ -1376,3 +1378,21 @@ def get_datasource_data(request, config_id, domain):
     query = cursor_based_query_for_datasource(request_params, datasource_adapter)
     data = response_for_cursor_based_pagination(request, query, request_params, datasource_adapter)
     return JsonResponse(data)
+
+
+@require_GET
+@api_auth()
+@requires_privilege_with_fallback(privileges.API_ACCESS)
+@api_throttle
+def get_cca_user_roles(request, domain):
+    if not toggles.SUPERSET_ANALYTICS.enabled(domain):
+        return HttpResponseForbidden(
+            _("This domain requires the superset-analytics flag to access this endpoint.")
+        )
+
+    try:
+        if request.method == 'GET':
```

Comment:
wont it always be GET since we have the decorator to ensure it?

Reply:
Good point. I copied this function and didn't remove the method check after I added the require_GET.